### ISSUE TITLE
[clsim] run the python tests (Trac #1252)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1246]
clsim has python tests under `resources/tests`, but cmake doesn't know about them.  Perhaps add `i3_test_scripts(resources/tests/*.py)` in the right place?

Migrated from https://code.icecube.wisc.edu/ticket/1252
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:45", 
    "description": "clsim has python tests under `resources/tests`, but cmake doesn't know about them.  Perhaps add `i3_test_scripts(resources/tests/*.py)` in the right place?", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067105393059", 
    "component": "combo simulation", 
    "summary": "[clsim] run the python tests", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-20T18:32:32", 
    "milestone": "", 
    "owner": "claudio.kopper", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150922T01:40:00: olivas changed milestone from "IceSim 4.2" to "IceSim 5.0"

**Comment by icecube-trac** [comment]
Trac update at 20151104T22:06:08: 
* claudio.kopper changed resolution from "" to "fixed"
* claudio.kopper changed status from "new" to "closed"
* claudio.kopper commented:

> In changeset 139193:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="139193"
> run python tests. closes #1246
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:45: 
* olivas changed milestone from "IceSim 5.0" to ""
* olivas commented:

> Milestone IceSim 5.0 deleted
